### PR TITLE
Added template for bump version workflow and docs.

### PR DESCRIPTION
**Pull request by jacobcrawford**

### CODE REVIEW

File: templates/github-workflows/bump_version_main.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        DEFAULT_BUMP: none
+        WITH_V: true
+        RELEASE_BRANCHES: main
+        INITIAL_VERSION: 0.0.0
+        DRY_RUN: false
+        TAG_CONTEXT: repo
+        PRERELEASE_SUFFIX: beta
+        VERBOSE: true
```
Comment (MartinSchmidt) [inline templates/github-workflows/bump_version_main.yaml:25]: Add newline

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]
Move to a seperate md file in the git folder.

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]
Move Git workflows to file in /git folder

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-# wiki
\ No newline at end of file
+# wiki
+
+### Git workflows
```
Comment (MartinSchmidt) [inline README.md:3]: - [ ] Move to new /git folder

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,9 @@
...
+The [**Version Bump**](templates/github-workflows/bump_version_main.yaml) workflow, bumps the version tag on a commit made to the main branch. The version bump depends on the content of the commit message:
+* Major (#major)
+* Minor (#minor)
+* Patch (#patch)
+
+This means that if you e.g. want to bump the version v1.0.0 to v1.0.1 your commit message to main should include the string '#patch'. There is no default version bump and therefore this workflow will fail if the commit message does not contain any of the three strings required to bump a specific version.
```
Comment (MartinSchmidt) [inline README.md:9]: something here test

- [ ] test

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,9 @@
-# wiki
\ No newline at end of file
+# wiki
+
```
Comment (MartinSchmidt) [inline README.md:2]: test here 1

- [ ] a
- [ ] b

File: README.md [hunk 5]
```diff
@@ -1 +1,9 @@
...
+### Git workflows
+The [**Version Bump**](templates/github-workflows/bump_version_main.yaml) workflow, bumps the version tag on a commit made to the main branch. The version bump depends on the content of the commit message:
+* Major (#major)
+* Minor (#minor)
+* Patch (#patch)
+
```
Comment (MartinSchmidt) [inline README.md:8]: test here b

- [ ] c
- [ ] d

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]
to review comment.

File: README.md (hunk 3 above)
Comment (skovbakke) [inline README.md:9]: Maybe add a paragraph on how to use the workflow in you project, i.e. where does the [.yaml file](https://github.com/distributed-technologies/wiki/pull/2/files#diff-bcf64a2380e5d1f5bd1f4b4d8eef3de98ea57164377d856c0d51b825c9a39025) go and what needs to be done (if anything) to make it work.

**Review by skovbakke — CHANGES_REQUESTED** [review]

File: git/workflows.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+#### Git workflows
```
Comment (MartinSchmidt) [inline git/workflows.md:1]:
```suggestion
# Git workflows
```

- [ ] I suggest changing the header, it is smaller than the version bumb header..

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]
Almost perfect 😅 I suggest changing the header size.

**Review by MartinSchmidt — APPROVED** [review]

**Review by skovbakke — APPROVED** [review]
Looks good

**Pull request merged**